gsutil.py

```diff
@@ -22,8 +22,8 @@
 
 # TODO: gsutil-beta: Distribute a pylint rc file.
 
-if not (2, 6) <= sys.version_info[:3] < (3,):
-  sys.exit('gsutil requires python 2.6 or 2.7.')
+# if not (2, 6) <= sys.version_info[:3] < (3,):
+#   sys.exit('gsutil requires python 2.6 or 2.7.')
```

Comment:
We should probably still have some requirements on the python version, like 2.7 or 3.4+.
```
if not ((2, 7) <= sys.version_info[:3] < (3,) or (3, 4) <= sys.version_info[:3] < (4,):
    sys.exit('gsutil requires python 2.7 or 3.4+.')
```

Reply:
Smart thinking! Or even a warning that we only support 2.7, >=3.5 and that unexpected behavior may occur.

Reply:
I think we should require the Python version to be ==2.7 or >=3.5, since that's all we officially develop for and test.

Reply:
Sounds good to me! I'll keep the blocker when we ship things. I'm not planning to merge this PR to master ever, it's just Kokoro testing,